src/index.js

```diff
@@ -287,6 +287,16 @@ class EliteDangerousJournalServer {
       socket.isReceiving = true;
     });
 
+    // handle socket errors
+    socket.on('error', (err) => {
+      // ignore that pesky ECONNRESET, socket will disconnect anyway, rethrow others
+      if (err.code && (err.code === 'ECONNRESET')) {
+        console.log(`${chalk.cyan('Socket')} ${chalk.red('Ignoring ECONNRESET')}`);
```

Comment:
I wonder if this should be a `console.error` or `console.warn` instead of `console.log` given that it is encountering an erro, but we're just sort of ignoring it.

Any thoughts, @rolandbosa?

Reply:
I'd prefer it to be completely muted, to be honest. There's no real information to be gained from this message. If you agree, I'll update the PR accordingly.

Reply:
Yeah, I could get behind something like:

```js
if (err.code !== 'ECONNRESET') {
  throw err;
}
```

Reply:
I went ahead and made the change and added it to the CHANGELOG so that I can get it published out before the weekend ends.

Thanks again for the PR! I look forward to seeing what you build with it.